src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/SecurityConfiguration.kt

```diff
@@ -60,7 +61,7 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
             val jwt = JwtDecoders.fromIssuerLocation(issuer).decode(userRequest.accessToken.tokenValue)
             val authorities = JSONArrayUtils.toStringList(jwt.claims["authorities"] as JSONArray?)
 
-            DefaultOAuth2User(authorities.stream().map { SimpleGrantedAuthority(it) }.toList(), user.attributes, "username")
+            DefaultOAuth2User(authorities.stream().map { SimpleGrantedAuthority(it) }.toList(), user.attributes, "name")
```

Comment:
This is referenced by the **sec:authentication="name"** in the **header.html** (see change below).